content/docs/concepts/multicluster-configuration/index.md

```diff
@@ -0,0 +1,101 @@
+
+### Single control plane topology
+
+This multicluster configuration uses a single Istio control plane,
+typically running on one of the clusters, whose Pilot
+manages services on both local and remote clusters and configures the Envoy sidecars
+for all of them. This approach works best in environments where all of the contributing
+clusters have VPN connectivity so every pod in the mesh is reachable from anywhere else using
+the same IP address.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-vpn.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters with direct network access to remote pods over VPN"
+    >}}
+
+In this configuration, one or more clusters run a remote Istio configuration which connects them
+to a single Istio control plane, allowing it to manage all of the Envoy's as a single mesh.
+Detailed instructions for setting up this kind of multicluster topology can be found
+in [single control plane with vpn](/docs/setup/kubernetes/multicluster-install/vpn/).
+
+If setting up a VPN network to provide universal pod-to-pod connectivity is difficult or impossible,
+Istio Pilot can be configured to enable location-aware service routing (a.k.a. split-horizon EDS).
+When configured this way, a request from a sidecar in one cluster to a service that is
+also running in the same cluster will be forwarded to the local service IP as usual, but when the destination
+workload is running in a different cluster it will use the remote cluster Gateway IP to connect to the service instead.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-split-horizon-eds.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using single Istio control plane and Gateway to reach remote pods"
+    >}}
+
+The [single control plane with gateways](/docs/examples/multicluster/split-horizon-eds/) example provides
+instructions for experimenting with this feature.
+
+### Multiple control plane topology
+
+Instead of using a central Istio control plane to manage the mesh,
+in this configuration each cluster has an **identical** Istio control plane
+installation, each managing its own endpoints.
+All of the clusters are under a shared administrative control for the purposes of
+policy enforcement and security.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-gateways.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using multiple Istio control planes and Gateway to reach remote pods"
+    >}}
+
+A single Istio service mesh across the clusters is achieved by replicating
+shared services and namespaces and using a common root CA in all of the clusters.
+Cross-cluster communication occurs over Istio Gateways of the respective clusters.
+
+Check out [multiple control plane with gateways](/docs/setup/kubernetes/multicluster-install/gateways/)
+for instructions on setting up this this kind of multicluster configuration.
```

Comment:
This text has a clear call-to-action, which is good, but it gives readers the contents at the end. Thus, readers could click the link without knowing whether or not it is really something they want to do. Consider reversing the clauses:
```suggestion
To set up this this kind of multicluster configuration, visit the [installation instructions for multiple control planes with gateways](/docs/setup/kubernetes/multicluster-install/gateways/).
```